deps/install.sh

```diff
@@ -71,6 +71,16 @@ esac
 # pull artifact from dependency repository
 curl -L --fail "${url}/${artifact}" > "${file}"
 
+shasum_command=()
```

Comment:
We also have [`bin/verify_hash.ps1`](https://github.com/runfinch/finch-core/blob/main/bin/verify_hash.ps1), should we be using that instead or should we delete it?

Reply:
Oh interesting perhaps it would be good to simplify down. Let me rework this to use that script instead.